client/src/components/ManageBookings/CurrentBooking/ChangeRequestStatus.tsx

```diff
@@ -0,0 +1,65 @@
+import Box from '@mui/material/Box';
+import Button from '@mui/material/Button';
+import Paper from '@mui/material/Paper';
+import { useState } from 'react';
+import { updateRequest } from '../../../helpers/APICalls/requests';
+import useStyles from './useStyles';
+interface props {
+  id: string;
+}
+
+const ChangeRequstStatus = ({ id }: props): JSX.Element => {
+  const [isSubmitting, setIsSubmitting] = useState(false);
+
+  const classes = useStyles();
+
+  const updateRequestToCancel = async () => {
+    setIsSubmitting(true);
+
+    const result = await updateRequest(id, 'cancelled');
+
+    window.location.href = '/my-jobs';
```

Comment:
Can we use React Router Dom hooks for redirecting rather than the window.location object

Reply:
Ditto throughout